src/main/java/movableInterface/MovableRectangle.java

```diff
@@ -0,0 +1,68 @@
+package movableInterface;
+
+public class MovableRectangle implements Movable {
+
+    private MovablePoint topLeft;
+    private MovablePoint bottomRight;
+
+    MovableRectangle(MovablePoint topLeft, MovablePoint bottomRight) {
+        this.topLeft = topLeft;
+        this.bottomRight = bottomRight;
+    }
+
+    @Override
+    public void moveUp() {
+        this.topLeft.moveUp();
+        this.bottomRight.moveUp();
+    }
+
+    @Override
+    public void moveDown() {
+        this.topLeft.moveDown();
+        this.bottomRight.moveDown();
+    }
+
+    @Override
+    public void moveLeft() {
+        this.topLeft.moveLeft();
+        this.bottomRight.moveLeft();
+    }
+
+    @Override
+    public void moveRight() {
+        this.topLeft.moveRight();
+        this.bottomRight.moveRight();
+    }
+
+    @Override
+    public String toString() {
+        return "Rectangle info: [topLeft " + this.getTopLeft() + ", bottomRight " + this.getBottomRight() + ", xSpeed: " + this.topLeft.getxSpeed() + ", ySpeed: " + this.bottomRight.getySpeed() + "]";
+    }
+
+    void print() {
```

Comment:
Such kind of methods are not related to entity itself.
For example, if you have some entity which should be populated from console, it should be done outside of the entity.
The same is for `print()` method. You have `toString()` to get String representation, so if you need to print it, you should do it in `Demo.class`, or somewhere else